sdk/core/azure-core/CMakeLists.txt

```diff
@@ -122,6 +124,12 @@ target_include_directories(azure-core PUBLIC ${CURL_INCLUDE_DIRS})
 target_include_directories(azure-core INTERFACE ${nlohmann_json_INCLUDE_DIRS})
 target_link_libraries(azure-core INTERFACE Threads::Threads nlohmann_json::nlohmann_json)
 
+if(NOT MSVC)
+    message("Find the OpenSSL package on MacOS, used for Base64 encoding.")
```

Comment:
It says MacOS but this message would come also on Linux.  Is that expected?

Reply:
I borrowed this logic from azure-storage's cmakelist and added this message, but it only seems to be required on MacOS. We seem to already be getting Openssl on linux (at least on the CI machines). But you are right, we will find openssl and link to azure-core even on linux. I will update the message later.

Reply:
It should be `macOS`, not `MacOS`